### DISCUSSION TITLE
I can not infer my mp4 video.

### DISCUSSION THREAD

**Discussion by Banboo-glich** [talmolab/sleap#793] [Help!] [answered]
Thank you for your creative work.    I want to use SLEAP in my graduation thesis.  

![20220614-121939](https://user-images.githubusercontent.com/65880626/173487689-366b42b5-17d7-46e4-8eda-13aaa338f7d5.jpg)
![20220614-122010](https://user-images.githubusercontent.com/65880626/173487693-3c644949-bdfa-4943-b625-1193922f630b.jpg)
 
 By the way, I guess the model training went well, but I can't infer my video.  Please tell me solution!

Best regards.

**Comment by talmo** [comment, marked as answer]
Hi @Banboo-glich,

It looks like you're trying to predict with just the centered instance model. Top-down models use both a centroid and centered instance sub-models that are stitched together to form a multi-stage network. We have some [more info on this in our docs](https://sleap.ai/guides/choosing-models.html#choosing-models).

You'll want to train a centroid model and pass it in as an extra argument (`-m path/to/centroid_model -m path/to/centered_instance_model`).

Give that a spin and let us know if you run into any issues!

Talmo

**Reply by Banboo-glich** [reply]
It went well! Tahnks!
